matlab_simulation/08-planning/shortestpath.m

```diff
@@ -1,4 +1,4 @@
-function [spath,sdist] = shortestpath(nodes, edges, start, finish)
+function [spath,sdist] = shortestpath(nodes, edges, start, finish, method, createVideo)
 %SHORTESTPATH Find shortest path using A-star search
```

Comment:
I think a key fact might be to include a heuristic (manhattan, euclidean etc) for the 'shortestpath' function according to Prof. Waslanders instructions. I think right now A-star its just using the euclidean distance